frontend/src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import "./Statistics.css";
```

Comment:
merge with master, now there is an index.css file there. Put your css' classes into this file and remove Statistics.css